contributions/catalog/README.md

```diff
@@ -0,0 +1,105 @@
+# Contribution-Catalog
+
+## Run in Development Mode
+MongoDB's service needs to be started and a Mongo Shell needs to be connected in a separate terminal instance
+
```

Comment:
It could be better if you let the user knows where (which folder) they can start.